.github/workflows/CI.yml

```diff
@@ -53,24 +53,27 @@ jobs:
         load: true
         cache-from: type=gha
         cache-to: type=gha,mode=max
+        outputs: type=docker,dest=/tmp/csz-bot.tar
+
+    - name: Upload image tarball
+      uses: actions/upload-artifact@v4
+      with:
+        name: csz-bot
+        path: /tmp/csz-bot.tar
```

Comment:
In anderen Projekten benutze ich docker/metadata-action:
https://github.com/docker/metadata-action

Die holt sich tags und labels automatisch aus den commits/tags/repo-infos:
```yaml
- id: meta
  uses: docker/metadata-action@v5
  with:
    images: ghcr.io/nulldev/csz-bot
    labels: |
      org.opencontainers.image.title=fluester-service
      org.opencontainers.image.vendor=pr0gramm.com

- name: Build & Push
  uses: docker/build-push-action@v6
  with:
    push: ${{ github.event_name != 'pull_request' }}
    tags: ${{ steps.meta.outputs.tags }}
    labels: ${{ steps.meta.outputs.labels }}
    platforms: linux/amd64
    cache-from: type=gha
    cache-to: type=gha,mode=max
```

Die setzt bei PRs auch ein Tag ans Image:
![image](https://github.com/user-attachments/assets/408f0073-c08f-435d-953b-8b2f5a837153)


kA wie man das Image dann hinterher aus der Registry wieder raus bekommt

Reply:
Ja, wenn's an die ephemeral envs geht, ist die nützlich